dali/imgcodec/util/convert.cc

```diff
@@ -26,6 +26,10 @@ namespace imgcodec {
 void Convert(SampleView<CPUBackend> out, TensorLayout out_layout, DALIImageType out_format,
              ConstSampleView<CPUBackend> in, TensorLayout in_layout, DALIImageType in_format,
              ROI roi, Orientation orientation) {
+  DALI_ENFORCE(in_layout.find('H') >= 0 && in_layout.find('W') >= 0,
+               "Input layout has to contain at least H and W dimensions.");
+  DALI_ENFORCE(out_layout.find('H') >= 0 && out_layout.find('W') >= 0,
+               "Output layout has to contain at least H and W dimensions.");
```

Comment:
I guess at this point it should be:
```suggestion
  int h_dim = ImageLayoutInfo::DimIndex(out_layout, 'H');
  int w_dim = ImageLayoutInfo::DimIndex(out_layout, 'W');
  DALI_ENFORCE(h_dim >= 0 && w_dim >= 0,
               "Output layout has to contain at least H and W dimensions.");
```
...and you could remove the
```
      auto h_dim = ImageLayoutInfo::DimIndex(out_layout, 'H');
      auto w_dim = ImageLayoutInfo::DimIndex(out_layout, 'W');
```
from the TYPE_SWITCH below.